### ISSUE TITLE
feature request: show check mark instead of inline results

### ISSUE THREAD

**Issue by marco-meer** [julia-vscode/julia-vscode#3424]
Instead of inline results, would it be possible to show only a check mark (as in line 50 in the screenshot) but show the whole result when hovering the mouse over it? Requesting this to have the option of a less crowded viewport.

![image](https://github.com/julia-vscode/julia-vscode/assets/99955854/44770e72-f24e-454a-8f53-7d59af156319)

**Comment by pfitzseb** [comment]
Yes, that's easy enough. Imo there should be some distinction between expandable results and genuine `nothing`s (which is what the check mark currently represents) though.
